sql/catseyexi-mods/CatsEyeXI-Add OupireSQL.sql

```diff
@@ -0,0 +1,104 @@
+
+-- Delete OLD Oupire Spell List
```

Comment:
should these lines only be processed once? remember this sql folder gets run on every fresh build of the server...

Reply:
the delete old is the LSB implemented spell/elemental list